### ISSUE TITLE
[Quick] question

### ISSUE THREAD

**Issue by thednp** [inexorabletash/polyfill#58]
Can I use the polyfill with Modernizr? I'm a bit confused as to what it actually does compared to your polyfill. I believe it's gonna be like this: I only need CSS selectors and CSS3 support, so only the first tab, your polyfill would probably include the HTML, DOM, EVENTS parts.

I just decided to go Native JS, I read left and right and feel very scared+amazed there is freedom.. so I speak a bit n00b-ish, sorry.

**Comment by inexorabletash** [comment]
Maybe? The way I build these polyfills usually follows this process:

* I'm making an addition to one of my hobby sites
* I realize it could benefit from using some newer part of the Web platform
* I try it out in a browser that's implemented it - great, it works!
* I try older browsers - oops, not working yet.
* Try writing a polyfill that adds the functionality. If that doesn't work, I scrap the idea and try a different approach.
* If it does work, I add it to these libraries.

That implies that I'm never striving for 100% coverage of the design, or to make it work seamlessly in all cases. I just need it to be "good enough" to fill in the parts that I needed for the original project. Also, if the polyfill complexity is too great, I can just scrap it and take a different approach.

Occasionally I write the polyfill not because I actually need it but because I want to explore the feature and see if I have feedback for the spec authors.

In any case, CSS is one area I'm not particularly obsessive about, because none of my sites hobby sites rely on anything style-related that won't degrade acceptably.

**Comment by thednp** [comment]
This answers my question in a pleasant way, I figured it on the way. You were not the only one don't really know what Modernizr is, so..

I was thinking the script in my site's head is gonna kick ass CSS3 engines and translations on IE7-9 out of the box, but reading about it, it only detects browser supported features. I never ever touched it for ever because I was thinking IE7-9 performance would do better than shadows or rounded corners. I always did some clean jQuery with some targeted CSS and I was alright.

I tested your code with some of my scripts, removed my ( attachEvent 'polyfill' ), replaced with yours, cleaned my code, only leave addEventListener and voila, IE8 could suddenly work with 'native' methods. Very nice.

BTW: I stumbled upon this and I am curious to know if you have something like this
http://www.thecssninja.com/javascript/handleevent
This is what I believe to be a $.one() equivalent, not sure.

Cheers
